_unittest/test_00_EDB.py

```diff
@@ -2946,14 +3005,57 @@ def test_147_find_dc_shorts(self):
         assert len(edbapp.nets["DDR4_DM3"].find_dc_short()) == 0
         edbapp.close()
 
-    def test_148_load_amat(self):
-        assert "Rogers RO3003 (tm)" in self.edbapp.materials.materials_in_aedt
-        material_file = os.path.join(self.edbapp.materials.syslib, "Materials.amat")
-        assert self.edbapp.materials.add_material_from_aedt("Arnold_Magnetics_N28AH_-40C")
-        assert "Arnold_Magnetics_N28AH_-40C" in self.edbapp.materials.materials.keys()
-        assert self.edbapp.materials.load_amat(material_file)
+    @patch("pyaedt.edb_core.materials.Materials.materials", new_callable=PropertyMock)
+    @patch.object(builtins, "open", new_callable=mock_open, read_data=MATERIALS)
+    def test_149_materials_read_materials(self, mock_file_open, mock_materials_property):
+        """Read materials from an AMAT file."""
+        mock_materials_property.return_value = ["copper"]
+        materials = Materials(MagicMock())
```

Comment:
Nice to see some `mock` around in PyAEDT.
I must confess I could have use some sometimes.
It is usually a good practice and could help us improve the test performance.